ngraph/core/include/ngraph/op/einsum.hpp

```diff
@@ -38,6 +38,12 @@ namespace ngraph
                 std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
 
+                /// \brief      Get an equation of Einsum operation
+                ///
+                /// \return     Einsum equation
+                ///
+                std::string get_equation() const { return m_equation; }
```

Comment:
Should we add `set_equation` method?

Reply:
We shouldn't. We have no use cases for it.